src/components/ButtonView/ButtonView.tsx

```diff
@@ -125,6 +131,8 @@ export const ButtonView: React.FC<Props> = ({
         styles.root,
         css`
           padding: ${padding}px;
+          height: ${height}px;
+          overflow-x: auto;
```

Comment:
I suppose the content should be scrollable by vertical due to buttons are shown in several lines

Reply:
updated at overflow-y